src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -248,4 +261,128 @@ namespace AppInstaller::CLI::Workflow
             std::filesystem::remove(path);
         }
     }
+
+    void MSStoreInstall(Execution::Context& context)
+    {
+        // Feature toggle check
+        if (!Settings::ExperimentalFeature::IsEnabled(Settings::ExperimentalFeature::Feature::ExperimentalMSStore))
+        {
+            context.Reporter.Error() << Resource::String::FeatureDisabledMessage << std::endl;

```

Comment:
>context.Reporter.Error() << Resource::String::FeatureDisabledMessage << std::endl; [](start = 12, length = 82)

This is typically coupled with the string that indicates which feature needs to be enabled.  I would suggest creating a Workflow object that can handle this generically.  So when invoking this from ExecuteInstaller:

context <<
    EnsureFeatureEnabled(Settings::ExperimentalFeature::Feature::ExperimentalMSStore) <<
    MSStoreInstall;

That way it can be reused easily.